frameworks/hdfs/tests/test_sanity.py

```diff
@@ -148,10 +147,23 @@ def test_kill_data_node():
 @pytest.mark.sanity
 @pytest.mark.recovery
 def test_kill_scheduler():
+    foldered_name = sdk_utils.get_foldered_name(config.SERVICE_NAME)
+    scheduler_name = sdk_utils.get_scheduler_name(foldered_name)
+    scheduler_id = sdk_tasks.get_task_ids("marathon", scheduler_name)
+    journal_ids = sdk_tasks.get_task_ids(foldered_name, "journal")
```

Comment:
Rather than breaking this up into three checks for each of the three types of nodes, this could just do a single non-prefixed retrieval of all nodes in the service, followed by a single check_tasks_not_updated below.

I think each call to check_tasks_not_updated involves a 30s wait to verify that no change occurs, so the test should finish a bit faster if only one check was performed. It would ultimately be equivalent to the three granular checks anyway.

Reply:
@nickbp remind me. The scheduler task doesn't show up in the list because it's a `marathon` task, correct?

Reply:
Yes

Reply:
Let's go with a single check then. Otherwise this looks good. Sorry for the back and forth Sam.

Reply:
For awhile I was wondering why `get_task_ids(foldered_name, scheduler_name)` wasn't returning an ID even though I saw it in the UI. Now it makes sense since it's a `marathon` task.

Anyway, I'm +1 for making it a single check.